core/storage/email/gae_models.py

```diff
@@ -338,19 +358,54 @@ def create(cls, user_id, exploration_id, thread_id):
 
     @classmethod
     def get_by_reply_to_id(cls, reply_to_id):
+        """Fetches the entry corresponding to the given 'reply-to' id.
+
+        Args:
+            reply_to_id: str. The unique 'reply-to' id.
+
+        Returns:
+            str or None. The entry corresponding to the given 'reply-to' id if
+                it is fetched else None.
+        """
         model = cls.query(cls.reply_to_id == reply_to_id).fetch()
         if not model:
             return None
         return model[0]
 
     @classmethod
     def get(cls, user_id, exploration_id, thread_id, strict=True):
+        """Gets the entry corresponding to the unique instance id.
+
+        Args:
+            user_id: str. The user id.
+            exploration_id: str. The exploration id.
+            thread_id: str. The thread id.
+            strict: bool. Whether to fail noisily if no entry with the given
+                instance id exists in the datastore. Default is True.
+
+        Returns:
+            An entry instance that corresponds to the given instance id if it is
+                present in the datastore. Otherwise, None.
+        """
         instance_id = cls._generate_id(user_id, exploration_id, thread_id)
         return super(
             FeedbackEmailReplyToIdModel, cls).get(instance_id, strict=strict)
 
     @classmethod
     def get_multi_by_user_ids(cls, user_ids, exploration_id, thread_id):
+        """Returns the entries corresponding to the given user ids in the dict
+        format.
+
+        Args:
+            user_ids: list(str). A list of user ids.
+            exploration_id: str. The exploration id.
+            thread_id: str. The thread id.
+
+        Returns:
+            dict. The enteries correponding to the given list of user ids in
```

Comment:
... in dict format. The key is the unique user ID, and the value is the FeedbackEmailReplyToIdModel instance corresponding to that user ID.

Though I'm confused by this. Can't there be multiple models for a given user ID? I think the docstring may be incorrect/incomplete.